msprime/mutations.py

```diff
@@ -213,25 +234,26 @@ def __init__(self):
 class HKYMutationModel(MatrixMutationModel):
     """
     The Hasegawa, Kishino and Yano mutation model (Hasegawa et al. 1985). Based on the
-    standard ACGT nucleotides as alleleic states, this model takes into account
+    standard ACGT nucleotides as alleleic states, this model allows different rates for
     transitions and transversions, and sets an equilibrium frequency for each nucleotide.
     In addition a custom ancestral frequency (``root_distribution``) can be specified.
     With ``kappa=1.0`` and the default values of the other arguments this model is equal
     to :class:`JC69MutationModel`. This model is similar to :class:`F84MutationModel`
     but with a differing parametrisation for ``kappa``.
 
+    See the documentation for more detail on the precise parameterization.
```

Comment:
There's quite a few places where it says to "see the documentation" Would be nice to say the section name (or URL?) as I'm not sure I'd know where to look otherwise.

Reply:
Right - I wasn't sure how to handle this, since on RTD we'd want to say "See below"; while in `help(msprime.HKYMutationModel)` we want to say "see https://msprime.readthedocs.io/en/latest/api.html#msprime.HKYMutationModel"... which on RTD would link to the very same place that the person is reading.

Alternatively, we could just put all of the details, that I've now got after the short description on RTD, into the docstring itself? How about that - or is that too much information?

Reply:
I think the second option, I don't have any issue with large docstrings, if they are all useful info.